test/api.scan.js

```diff
@@ -252,6 +262,56 @@ describe('picomatch', () => {
         negated: false
       });
     });
+
+    it('should return parts of the pattern', () => {
+      // Right now it returns []
+      // assertParts('', ['']);
+      // assertParts('*', ['*']);
+      // assertParts('.*', ['.*']);
+      // assertParts('**', ['**']);
+      // assertParts('foo', ['foo']);
+      // assertParts('foo*', ['foo*']);
+      // assertParts('/', ['', '']);
+      // assertParts('/*', ['', '*']);
+      // assertParts('./', ['']);
+      // assertParts('{1..9}', ['{1..9}']);
+      // assertParts('c!(.)z', ['c!(.)z']);
+      // assertParts('(b|a).(a)', ['(b|a).(a)']);
+      // assertParts('+(a|b\\[)*', ['+(a|b\\[)*']);
+      // assertParts('@(a|b).md', ['@(a|b).md']);
+      // assertParts('(a/b)', ['(a/b)']);
+      // assertParts('(a\\b)', ['(a\\b)']);
+      // assertParts('foo\\[a\\/]', ['foo\\[a\\/]']);
+      // assertParts('foo[/]bar', ['foo[/]bar']);
+      // assertParts('/dev\\/@(tcp|udp)\\/*\\/*', ['', '/dev\\/@(tcp|udp)\\/*\\/*']);
+
+      // Right now it returns ['*']
+      // assertParts('*/', ['*', '']);
+
+      // Right now it returns ['!(!(bar)', 'baz)']
+      // assertParts('!(!(bar)/baz)', ['!(!(bar)/baz)']);
```

Comment:
@mrmlnc are the commented out tests still passing?

Reply:
These tests (here and above) have an incorrect result for the current implementation of the `scan` method.

I can delete them or move them to another `it` with `skip` or something.